packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -742,6 +751,12 @@ export class PipelineEditor extends React.Component<
     });
   }
 
+  handleOpenRuntimes(): void {
+    this.app.shell.activateById(
+      `elyra-metadata:${RUNTIMES_NAMESPACE}:${KFP_SCHEMA}`
```

Comment:
It is overly specific, I used this naming for Runtimes so it matched the generic naming created when using `DefaultMetadataWidget`, meaning if the `elyra-metadata:open` command was called with the `runtimes` namespace and `kfp` schema it would open the Runtimes UI rather than creating a new generic one that shows the json dump rather than the links.

I'm not particularly attached to this idea though, its mostly there as a (cleaned up) artifact of my dev process rather than an intentional design.